metadata-io/build.gradle

```diff
@@ -91,6 +93,10 @@ test {
   testLogging.exceptionFormat = 'full'
 }
 
+repositories {
```

Comment:
Curious why this is needed, shouldn't this inherit from the root?

Reply:
hmmm, shouldn't be